scheduler/src/cook/mesos/api.clj

```diff
@@ -1369,27 +1370,51 @@
       (when-let [uncoerced-query-params (get-in ctx [:request :query-params "job"])]
         (mapv #(UUID/fromString %) (vectorize uncoerced-query-params)))
       (get-in ctx [:request :body-params :jobs])
-      [(get-in ctx [:request :body-params :job])]))
+      (vectorize (get-in ctx [:request :body-params :job]))))
 
 (defn check-jobs-exist
   [conn ctx]
-  (let [jobs (jobs-from-request ctx)
+  (let [jobs (or (::jobs ctx) (jobs-from-request ctx))
```

Comment:
Should we instead modify `jobs-from-request`?

Reply:
I like that a lot better. Done.